jobs/gtfs-rt-parser-v2/Dockerfile

```diff
@@ -3,23 +3,23 @@ FROM openjdk:11
 LABEL org.opencontainers.image.source https://github.com/cal-itp/data-infra
 
 ENV GTFS_RT_VALIDATOR_JAR=/gtfs-realtime-validator.jar
-ENV GTFS_VALIDATOR_VERSION=v1.0.0
+ENV GTFS_RT_VALIDATOR_VERSION=v1.0.0
 
 RUN apt-get update -y \
-    && apt-get install -y python3 python3-pip
+    && apt-get install -y python3 python3-pip python3-venv
 
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
-ENV PATH="${PATH}:/root/.poetry/bin"
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3 -
+ENV PATH="/root/.local/bin:${PATH}"
 
-# formerly the "1.0.0-SNAPSHOT" from S3
-COPY ./rt-validator-v1.0.0_cli.jar ${GTFS_RT_VALIDATOR_JAR}
+# from https://github.com/MobilityData/gtfs-realtime-validator/packages/1268973
+COPY ./gtfs-realtime-validator-lib-1.0.0-20220223.003525-2.jar ${GTFS_RT_VALIDATOR_JAR}
```

Comment:
I am a little 👀 at this.... are we sure that this is the same file?

Reply:
no but it's the latest from the repo artifacts, I figure now is the time to grab it and document the URL :)

Reply:
ok. on a very pedantic level I think that technically puts us in the same place for RT as schedule where the new pipeline has a diff validator version. given the limited visibility of RT validations thus far probably ok but _technically_ I think we are introducing a change here